packages/react-magma-dom/src/components/Dropdown/DropdownDivider.tsx

```diff
@@ -17,8 +25,17 @@ const StyledHr = styled.hr`
 export const DropdownDivider: React.FunctionComponent<DropdownDividerProps> = (
   props: DropdownDividerProps
 ) => {
-  const { testId, ...other } = props;
+  const { isInverse, testId, ...other } = props;
 
   const theme = React.useContext(ThemeContext);
-  return <StyledHr {...other} data-testid={testId} theme={theme} />;
+  const context = React.useContext(DropdownContext);
+
+  return (
+    <StyledHr
+      {...other}
+      data-testid={testId}
+      isInverse={context.isInverse}
```

Comment:
Should `isInverse` come from context or props? If context, we don't need the change in line 28.

Reply:
Context, it was removed from 28 on the previous commit.